Dockerfile

```diff
@@ -127,6 +127,10 @@ RUN /root/.bitrise/tools/stepman update
 
 COPY ./ssh/config /root/.ssh/config
 
+# ------------------------------------------------------
+# --- Install Rubygems
+
+RUN gem update --system --no-document
```

Comment:
Yep `"can't modify frozen String"` is why I submitted this PR. 😄 

I've updated the wording to say `Upgrade`